Contributing.md

```diff
@@ -2,38 +2,32 @@
 ---
 ## Want to help? We would love that!
 Here are some ways to get involved, ranging in
-difficulty from easiest to hardest.
+difficulty from easiest to hardest
 
 ## Bug Hunting
 Run the software and tell us when it breaks. We are happy to receive bug
-reports. This software was developed for internal use
-on the fly as needed. This means that the code was not built to the
-typical standards of an open source project, but we would like to see it get
-there.
+reports
 
 Just be sure to do the following:
 * Check if the bug is already accounted for on the
 [Github issue tracker](https://github.com/activecm/rita/issues)
-  * If an issue already exists, add the following info in a comment
-  * If not, create an issue, and include the following info
+  * If an issue already exists, add the relevant info in a comment
+  * If not, create an issue and include the relevant info
 * Give very specific descriptions of how to reproduce the bug
   * Log files can be found at ~/.rita/logs
```

Comment:
This is no longer true. It's determined by the setting in the config file with the default being `/var/lib/rita/logs`. But how about we just delete this line?

Reply:
Sounds good.